### ISSUE TITLE
leading whitespace on element with embedded newlines

### ISSUE THREAD

**Issue by jmls** [mozilla/nunjucks#1096]
if I have this template, with `autoescape` set to true

```
<body>
    <div>
           {{ notes }}
    </div>
</body>
```
and `notes` has  embedded linefeeds, then the output is like this

```
<body>
    <div>
        line1
line2
line3
    </div>
</body>
```
is there any option / technique to make the `line2` and `line3` line up with `line1` ?

**Comment by ArmorDarks** [comment]
No, as it doesn't make any sense. Final HTML should be minified before putting it into production anyway.
